### PR TITLE
[ch3212] fix display + wording + link admin

### PR DESCRIPTION
**Pull request by cacahouwete**
https://app.clubhouse.io/enmarche/story/3212/cr%C3%A9er-dans-l-espace-r%C3%A9f%C3%A9rent-une-nouvelle-tab-%C3%A9quipe-d%C3%A9partementale-qui-pr%C3%A9sentera-l-organigramme

### CODE REVIEW

File: templates/referent/organizational_chart_macros.html.twig [hunk 1]
```diff
@@ -1,39 +1,47 @@
...
                         <br />
+                        <span>
                         {% set ref_person_link = get_ref_person_link(item, referent) %}
                         {% if ref_person_link %}
                             {{ ref_person_link.firstName }} {{ ref_person_link.lastName }}
                         {% else %}
                             -
                         {% endif %}
+                    </span>
                         {% if editable %}
```
Comment (ottaviano) [inline templates/referent/organizational_chart_macros.html.twig]: juste if `editable`?

File: templates/referent/organizational_chart_macros.html.twig [hunk 2]
```diff
@@ -1,39 +1,47 @@
...
             </li>
         {% endfor %}
     </ol>
 {% endmacro %}
 
-{% macro orga_chart_item(item, referent, editable) %}
+{% macro orga_chart_item(item, referent, editable, admin) %}
     {% import _self as oc %}
     <div>
         <h{{ item.level }}>
```
Comment (ottaviano) [inline templates/referent/organizational_chart_macros.html.twig]: are you check the range of level `[1-6]`?

File: templates/referent/organizational_chart_macros.html.twig [hunk 3]
```diff
@@ -1,39 +1,47 @@
...
                             {{ ref_person_link.firstName }} {{ ref_person_link.lastName }}
                         {% else %}
                             -
                         {% endif %}
+                    </span>
                         {% if editable %}
                     </a>
                 {% endif %}
+            {% elseif admin %}
+                <a href="{{ admin ? admin.generateObjectUrl('edit', item, admin.listFieldDescriptions['batch'].options.route.parameters) : path('app_referent_referent_person_link_edit', {id: item.id}) }}">
```
Comment (ottaviano) [inline templates/referent/organizational_chart_macros.html.twig]: double check of `admin` value

File: templates/referent/organizational_chart_macros.html.twig (hunk 2 above)
Comment (cacahouwete) [reply templates/referent/organizational_chart_macros.html.twig]: no :s

File: templates/referent/organizational_chart_macros.html.twig [hunk 4]
```diff
@@ -1,39 +1,47 @@
...
+{% macro orga_chart_item(item, referent, editable, admin) %}
     {% import _self as oc %}
     <div>
         <h{{ item.level }}>
             {% if item.referentPersonLinks is defined %}
-                {% if editable %}
-                    <a href="{{ path('app_referent_referent_person_link_edit', {id: item.id}) }}">
+                {% if editable or admin %}
+                    <a href="{{ admin ? admin.generateObjectUrl('edit', item, admin.listFieldDescriptions['batch'].options.route.parameters) : path('app_referent_referent_person_link_edit', {id: item.id}) }}">
                         {% endif %}
```
[the remark this answers is not in the document]
Comment (cacahouwete) [reply templates/referent/organizational_chart_macros.html.twig]: why ?

[the remark this answers is not in the document]
Comment (ottaviano) [reply templates/referent/organizational_chart_macros.html.twig]: 👍

File: templates/referent/organizational_chart_macros.html.twig (hunk 1 above)
Comment (ottaviano) [reply templates/referent/organizational_chart_macros.html.twig]: @cacahouwete you open `<a>` tag if editable or admin, but you close it only if editable.

**Comment by ottaviano** [comment]
@cacahouwete, it'll be a little bug: 
https://github.com/EnMarche/en-marche.fr/pull/2351#comment-181409420

**Pull request merged**